### PR TITLE
Update cicd Troubleshooting.md

### PR DESCRIPTION
**Pull request by jesp209i**
## Description

Updates to troubleshooting

## Type of suggestion

* [ ] Typo/grammar fix
* [x] Updated outdated content
* [ ] New content
* [ ] Updates related to a new version
* [ ] Other

## Product & version (if relevant)

Cloud / CICD Flow

### CODE REVIEW

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 1]
```diff
@@ -84,12 +84,20 @@ You can see an example of how you could zip your repository before uploading it,
...
-The error tells you which package to look for and which version is currently in your left-most cloud environment.
+```
+Cannot apply update because the following packages would be downgraded: Package: Umbraco.Cms, Version: 13.4.0 ==> Project file: /Update/src/MyAwesomeProject.Code/MyAwesomeProject.Code.csproj contains lower Version: 13.1.0 .
+```
+
+The error tells you which package to look for and which version is currently in your left-most cloud environment. The error also contains the problematic incoming .csproj-file and the version referenced by it.
+
+If there are referenced multiple packages with lower versions in the incomming package, the error will list each.
+
+We recommend that you align versions of the mentioned package(s) in your csproj-files to the higher version mentioned in the error for that package or a later version.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:97]:
```suggestion
We recommend aligning the package versions in your _.csproj_ files with the higher version mentioned in the error for that package or a later version.
```

File: umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md [hunk 2]
```diff
@@ -84,12 +84,20 @@ You can see an example of how you could zip your repository before uploading it,
...
+
+The error tells you which package to look for and which version is currently in your left-most cloud environment. The error also contains the problematic incoming .csproj-file and the version referenced by it.
+
+If there are referenced multiple packages with lower versions in the incomming package, the error will list each.
+
+We recommend that you align versions of the mentioned package(s) in your csproj-files to the higher version mentioned in the error for that package or a later version. 
 
 If you have orphaned csproj-files you should remove them or rename them. 
-Orphaned would be backup csproj files or files not referenced by the any of the main project-files.
+Orphaned would be backup csproj files or files not referenced by the any of the main project-files nor referenced in a .sln-file.
```
Comment (eshanrnh) [inline umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md:100]:
```suggestion
Orphaned would be backup _.csproj_ files or files not referenced by any of the main project files nor referenced in a _.sln_ file.
```

**Review by eshanrnh — CHANGES_REQUESTED** [review]
Thanks for the PR, @jesp209i 🙌 I've added some suggestions to clarify the sentences and fix the vale error.

Can you take a look and let me know if it doesn't look right?

**Comment by jesp209i** [comment]
Great suggestions @eshanrnh 👍 
All suggestions accepted - thanks! 🌞

**Review by eshanrnh — APPROVED** [review]
Looks good to me, @jesp209i 🙌

**Pull request merged**